runtime/src/main/java/org/corfudb/runtime/CheckpointWriter.java

```diff
@@ -239,10 +243,15 @@ public long finishCheckpoint() {
         return endAddress;
     }
 
-    public static long startGlobalSnapshotTxn(CorfuRuntime rt) {
-        TokenResponse tokenResponse =
-                rt.getSequencerView().nextToken(Collections.EMPTY_SET, 0);
-        long globalTail = tokenResponse.getToken().getTokenValue();
+    public static long startGlobalSnapshotTxn(CorfuRuntime rt, Long forceAddress) {
```

Comment:
If this method depends on a member field, do we still want this method to be static?